### PR TITLE
Made check incase PyTorch didn't define the device

### PR DESCRIPTION
**Pull request by bathmatt**
**Summary**

minor change if the PyTorch model wasn't written for cuda.
**Author(s)**

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Backward Compatibility**

**Implementation Notes**

**Post Submission Checklist**

- [x] The feature or features in this pull request is complete
- [x] Licensing information is complete
- [x] Corresponding author information is complete
- [x] The source code follows the LAMMPS formatting guidelines
- [ ] Suitable new documentation files and/or updates to the existing docs are included
- [ ] The added/updated documentation is integrated and tested with the documentation build system
- [ ] The feature has been verified to work with the conventional build system
- [x] The feature has been verified to work with the CMake based build system
- [ ] Suitable tests have been added to the unittest tree.
- [ ] A package specific README file has been included or updated
- [ ] One or more example input decks are included

**Further Information, Files, and Links**

### CODE REVIEW

**Comment by bathmatt** [comment]
@lubbersnick just a heads up, this is a minor change.

File: python/lammps/mliap/pytorch.py [hunk 1]
```diff
@@ -132,22 +133,26 @@ def forward(self, elems, descriptors, beta, energy,use_gpu_data=False):
...
 
             if (use_gpu_data):
-                energy_nn = torch.as_tensor(energy,dtype=self.dtype, device=self.device)
+                energy_nn = torch.as_tensor(energy,dtype=self.dtype, device=device)
                 energy_nn[:] = self.model(descriptors, elems).flatten()
             else:
                 energy_nn = self.model(descriptors, elems).flatten()
                 energy[:] = energy_nn.detach().cpu().numpy().astype(np.float64)
-
+
```
Comment (stanmoore1) [inline python/lammps/mliap/pytorch.py]: Needs whitespace cleanup (remove trailing spaces) on line 147

**Review by stanmoore1 — CHANGES_REQUESTED** [review]
Looks OK to me, but needs whitespace cleanup

File: python/lammps/mliap/pytorch.py (hunk 1 above)
Comment (akohlmey) [reply python/lammps/mliap/pytorch.py]: Just fixed this

**Comment by lubbersnick** [comment]
> @lubbersnick just a heads up, this is a minor change.

Looks sensible to me!

**Review by sjplimp — APPROVED** [review]

**Review by stanmoore1 — APPROVED** [review]
I approve

**Pull request merged**